src/pages/signin/ResendValidationForm.js

```diff
@@ -139,6 +139,7 @@ class ResendValidationForm extends React.Component {
                         </Text>
                     </TouchableOpacity>
                     <Button
+                        medium
```

Comment:
Just curious - if you don't specify medium, what size is the default? I wonder if we want to make medium the default size if it's not already?

Reply:
It takes `styles.button` which has the height of `large` size with `ph3` horizontal spacing.

Reply:
This ambiguity would also be removed if we had a `size` prop with a default value 😏
Not going to address here but in https://github.com/Expensify/App/issues/7220 instead